hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationEndpoint.java

```diff
@@ -296,6 +300,53 @@ public boolean evaluate() throws Exception {
     hbaseAdmin.removeReplicationPeer("testWALEntryFilterFromReplicationEndpoint");
   }
 
+  @Test
+  public void testNamespacesMutualExclusiveScopesWALEntryFilter() throws Exception {
+    Set<String> namespaces = new HashSet<String>();
+    namespaces.add("default");
+    ReplicationPeerConfig rpc = ReplicationPeerConfig.newBuilder()
+      .setClusterKey(ZKConfig.getZooKeeperClusterKey(CONF1))
+      .setReplicationEndpointImpl(SelfWrappedReplicationEndpointForTest.class.getName())
+      .setReplicateAllUserTables(false)
+      // sets namespaces
+      .setNamespaces(namespaces).build();
+    hbaseAdmin.addReplicationPeer("testNamespacesMutualExclusiveScopesWALEntryFilter", rpc);
```

Comment:
mind making `testNamespacesMutualExclusiveScopesWALEntryFilter` a constant string in the test?

Reply:
Switched to `TestName` usage.